core/state/pevm_statedb.go

```diff
@@ -486,9 +486,9 @@ func (pst *UncommittedDB) Merge(deleteEmptyObjects bool) error {
 		// so we don't need to merge anything.
 		return nil
 	}
-	if err := pst.conflictsToMaindb(); err != nil {
-		return err
-	}
+	//if err := pst.conflictsToMaindb(); err != nil {
+	//	return err
+	//}
```

Comment:
we can keep this uncommented, and make separate PR later for this when it is confirmed to be OK